src/jit/function-builder.cc

```diff
@@ -726,29 +749,22 @@ bool FunctionBuilder::Emit(TR::BytecodeBuilder* b,
     }
 
     case Opcode::LocalGet: {
-      Type t;
-      uint32_t off = GetLocalOffset(&stack, &t, interp::ReadU32(&pc));
-
-      if (t == Type::V128)
-        return false;
-
-      stack.Push(b->LoadIndirect("Value", TypeFieldName(t), PickPhys(b, off)));
+      uint32_t local_depth = interp::ReadU32(&pc);
+      stack.Push(stack.Pick(local_depth - 1));
```

Comment:
While playing around with this PR for performance testing purposes, I realized there's a bug here. The `IlValue*` being pushed onto the stack here needs to be a _copy_ of the one we pick from the stack since the value of the local may change between the time that the `local.get` opcode appears and when the value is used.